pkg/skaffold/docker/auth.go

```diff
@@ -77,9 +77,7 @@ func (credsHelper) GetAllAuthConfigs() (map[string]types.AuthConfig, error) {
 		return nil, errors.Wrap(err, "docker config")
 	}
 
-	// TODO(dgageot): this is really slow because it has to run all the credential helpers.
-	// return cf.GetAllCredentials()
```

Comment:
i think the reason why this was commented out is because it is slow.

Reply:
I mentioned this in the PR summary - while it is slower than not getting all credentials, it would also mean that the image build is not broken if it needs to pull from a private repo. It would arguably be better to be slower but functional for more use cases. 

Additionally it should be no slower than a ‘docker build’.